### ISSUE TITLE
Default argument in template definition is being wiped out

### ISSUE THREAD

**Issue by ajusa** [geekrelief/genit#6]
```nim
gen +Tag:
  template `it[0]`(attrs: openarray[(string, string)] = default(array[0, (string, string)]), inner: untyped = dummy()): untyped =
    node(Tag.`it[0]`, attrs, inner)
```
output
```nim
    template `menu`(attrs: openarray[(string, string)] = default(array[0]);
                    inner: untyped = dummy()): untyped =
      node(Tag.`menu`, attrs, inner)
```
Notice how for the attrs argument, the (string, string) bit is wiped out by genit and doesn't make it to the generated code.

**Comment by geekrelief** [comment]
Thanks for reporting. Can you provide a small sample I can use to test/repro this?

**Comment by geekrelief** [comment]
nevermind, I can repro this. I'll have a fix soon.

**Comment by geekrelief** [comment]
This is fixed in 0.13.1.  It should resolve the issue.  Let me know if it doesn't work for you.

**Comment by ajusa** [comment]
Thanks for the prompt fix!
